### ISSUE TITLE
Investigate why does the checkbox label repeat twice in BASE?

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#330]
As stated in issue #322, I hear the Two-balloon experiment label twice.

I do not know why.

1. Is this normal behaviour for a checkbox with an associated label?
2. Could it be due to order? Should the label come first?
3. Could it be due to the extra div wrappers?
4. Would using an `aria-label `attribute remove the repetition on a visually  unlabeled checkbox?

**Comment by jessegreenberg** [comment]
@terracoda for reference, here is the HTML for this checkbox now:

```html
<div tabindex="-1" id="container-421-482-716-792-783-766">
    <input id="421-482-716-792-783-766" tabindex="0" type="checkbox" style="width: 1px;">
    <label tabindex="-1" id="label-421-482-716-792-783-766" for="421-482-716-792-783-766">Two-Balloon Experiments</label>
    <p tabindex="-1" id="description-421-482-716-792-783-766">Play with two balloons or just one.</p>
</div>
```

**Comment by terracoda** [comment]
Thanks. I will review it and see if I can figure out why the label repeats :-)

**Comment by jessegreenberg** [comment]
@terracoda is this something that you would like to work on for this version of BASE?

**Comment by terracoda** [comment]
@jessegreenberg, thanks for asking. This is a longterm general question which I haven't been able to get to yet.

**Comment by terracoda** [comment]
@jessegreenberg, I created some checkbox examples in the html-sketches folder. It seems the label text repeats when the label is associated with the actual checkbox with the for attribute. At least this is how it works on VoiceOver. 

I really do not know how this repetition improves the experience for screen reader users!

The label text does not repeat when I use the `aria-label` attribute!

I am going to test a nested input now, too, which should be fine because in that case I think the for attribute is not required. I would want to double check that though.

**Comment by terracoda** [comment]
Label text does not repeat when the `input` is nested! No label text repetition with or without the `for` attribute.

Can the nested input be our new inclusive design pattern?

**Comment by terracoda** [comment]
@jessegreenberg, @zepumph, @mbarlow12, what do you think? Can the nested` input` be our new inclusive design pattern? I think Scenery supports this nesting now, right?

Please listen to the checkbox examples in html-sketches.

**Comment by terracoda** [comment]
I added to the A11y Design meeting agenda to discuss on Friday and potential further action.

**Comment by jhung** [comment]
@jobara - I recall having an issue similar to this in one of our components. Do you recall what that was and how we fixed it?

**Comment by jessegreenberg** [comment]
Meeting notes: @terracoda discovered that there is no repetition when the aria-label attribute is used, and that label sounds best on VO. Scenery currently does not support adding HTML and innerText as a child of an element so we will proceed with an aria-label.

**Comment by jessegreenberg** [comment]
@terracoda we changed it to use aria-label instead of a label element. I tested with NVDA and VO and it is sounding nice. Could you please verify that this is working? Is there anything else that needs to be done for this issue?

**Comment by terracoda** [comment]
@jessegreenberg, thanks. I am closing this issue.
